code/game/turfs/floor.dm

```diff
@@ -490,10 +490,10 @@ GLOBAL_LIST_INIT(wood_icons, list("wood", "wood-broken"))
 		levelupdate()
 		playsound(src, 'sound/weapons/genhit.ogg', 25, 1)
 
-	else if(istype(I, /obj/item/tool/shovel))
-		if(!is_grass_floor())
-			to_chat(user, "<span class='warning'>You cannot shovel this.</span>")
-			return
+	//else if(istype(I, /obj/item/tool/shovel))
+		//if(!is_grass_floor())
+			//to_chat(user, "<span class='warning'>You cannot shovel this.</span>")
+			//return
```

Comment:
?

Reply:
Its the code saying that "you cannot shovel this", but in my testing, it only works on GRASS. Like the grass tile itself. Not dirt or sand. I don't see a use for it as it is just code for that message

Reply:
ok then delete altogether if you think it's useless